<launch>
 <group ns="lidar1">
  <node name="lidar01" pkg="bluesea2" type="bluesea2_node" output="screen" >
    <param name="type" value="uart" />
    <param name="platform" value="LDS-50C-2" />
    <param name="port" value="/dev/ttyUSB0" />
    <rosparam param="rate_list">[ 500000, 768000, 921600 ]</rosparam>
    <param name="baud_rate" value="230400" />
    <param name="max_dist" value="50.0"/>
    <param name="raw_bytes" value="3"/>
    <param name="frame_id" value="map" />
    <param name="output_360" value="true" />
    <param name="output_scan" value="true" />
    <param name="output_cloud" value="false"/>
    <param name="with_angle_filter" value="false"/>
    <param name="min_angle" value="-2.2"/>
    <param name="max_angle" value="2.2"/>
    <param name="max_dist" value="50.0"/>
    <param name="hard_resample" value="false"/>
    <param name="soft_resample" value="true"/>
    <param name="resample_res" value="1"/>
    <param name="unit_is_mm" value="true"/>
    <param name="with_confidence" value="true"/>
    <param name="with_checksum" value="true"/>
    <param name="from_zero" value="true"/>
    <param name="inverted" value="false"/>
    <rosparam param="mask1">[ -1.5, 0 ]</rosparam>
    <param name="Savelog" value="false"/>
    <param name="logPath" value="/home/pacecat/bluesea/test.log"/>
  </node>
 </group>
</launch>


